src/radical/pilot/task_overlay/worker.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, cfg, session=None):
         # FIXME: this should be delegated to ru.generate_id
 
         # FIXME: why do we need to import `os` again after MPI Spawn?
-        import os
+        import os                                                         # noqa
```

Comment:
Line 2 imports os. There is no need to import it here.

Reply:
For some reason,  the `os` module was not in the namespace after fork/exec.  Don't ask, but thus the import and the comment...